<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePwd</class>
 <widget class="QWidget" name="ChangePwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>修改密码</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 173);</string>
  </property>
  <widget class="QLabel" name="hint_lbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入原密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="confirm_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldpwd_let">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hint_lbl1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请输入新密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="newpwd_let1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="newpwd_let2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>280</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="hint_lbl2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>再次确认新密码</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="oldPwd_hint">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="newPwd_hint_1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="newPwd_hint_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>280</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hint_lbl_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>方正小标宋简体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 224, 102);</string>
   </property>
   <property name="text">
    <string>密码长度为6-18位，由数字、字母和下划线组成</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
